config/webpack.config.dev.js

```diff
@@ -14,6 +14,26 @@ var HtmlWebpackPlugin = require('html-webpack-plugin');
 var CaseSensitivePathsPlugin = require('case-sensitive-paths-webpack-plugin');
 var paths = require('./paths');
 
+// We don't watch node_modules by default because it's too slow.
+// However if a dependency is missing, and the user npm installs it,
+// we want to rebuild the app and disregard the cached "module not found".
+// This custom plugin does the job.
+// Relevant issue: https://github.com/facebookincubator/create-react-app/issues/186
+function RebuildOnPackageJSONChangePlugin(packageJsonPath) {
```

Comment:
Sort of, I wouldn't worry about the for now though. I don't want people to start depending on it or think it's a "best practice". Ideally we shouldn't need it because there should be a better way to fix this.